pytroll_collectors/geographic_gatherer.py

```diff
@@ -73,17 +73,26 @@ def _clean_config(self):
                 raise NoOptionError
 
     def _setup_publisher(self):
+        publisher_config = self._collect_publisher_config()
+        self.publisher = create_publisher_from_dict_config(publisher_config)
+        self.publisher.start()
+
+    def _collect_publisher_config(self):
         if self._opts.config_item:
             publisher_name = "gatherer_" + "_".join(self._opts.config_item)
         else:
             publisher_name = "gatherer"
 
         publish_port = self._opts.publish_port
         publisher_nameservers = self._opts.nameservers
-
-        self.publisher = publisher.NoisyPublisher(publisher_name, port=publish_port,
-                                                  nameservers=publisher_nameservers)
-        self.publisher.start()
+        if publisher_nameservers is not None:
+            publisher_nameservers = False if 'false' in publisher_nameservers else publisher_nameservers
```

Comment:
I'd like to have just a little more error checking here: if the number of nameservers is not one, when one of them is "false", then I think an error is in order.

Reply:
Umm, not sure how to read that. So there should be an error in this case?
```python
nameservers = ['localhost', 'false']
```